modules/ROOT/partials/install/upgrading-info.adoc

```diff
@@ -1,3 +1,4 @@
+[[upgradingtothelatestversionoftinymce5]]
 == Upgrading to the latest version of TinyMCE 5
 
```

Comment:
I can't leave a comment on the exact line, but below the `<VERSION>` marker isn't rendering correctly as it's using:
```
_`<VERSION>`_
```
and instead needs to be this:
```
`_<VERSION>_`
```

Reply:
done